website/src/components/CollapsableText.tsx

```diff
@@ -29,6 +29,7 @@ export const CollapsableText = ({
   if (typeof text !== "string" || text.length <= maxLength) {
     return <>{text}</>;
   } else {
+    const text_with_breaks = text.split("\n").map((str, index) => <p key={index}>{str}</p>);
```

Comment:
I feel like there's gotta be a less strange way to handle this.  Have you looked into alternatives?